app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/repository/repositoryBranchesComposable.kt

```diff
@@ -0,0 +1,27 @@
+package com.bottlerocketstudios.brarchitecture.ui.repository
```

Comment:
@LuisRocketRangel Refactor the filename to be UpperCamelCased, `RepositoryBranchesComposable.kt`. Leave the extension function as is :)

Reply:
Fixed!